docs/management_api.yml

```diff
@@ -366,6 +366,105 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /settings/tokens:
+    get:
+      operationId: List User Personal Access Tokens
+      tags:
+        - Management API
+      security:
+        - ManagementJWT: []
+      summary: Get user Personal Access Tokens
+      responses:
+        200:
+          description: Successful response - a list of tokens is returned.
```

Comment:
nit: passive, according to the docs guidelines we avoid passive voice. suggested change: `"Endpoint returns a list of tokens"`

Reply:
fixed